action.yml

```diff
@@ -1,14 +1,20 @@
 # action.yml
-name: "Codacy Coverage Reporter"
-description: "Multi-language coverage reporter for Codacy"
+name: 'Codacy Coverage Reporter'
+description: 'Multi-language coverage reporter for Codacy'
 branding:
   icon: 'check'
   color: 'gray-dark'
 inputs:
   project-token:
+    description: 'Project token for the Codacy project you want to send coverage information'
     required: true
+  coverage-reports:
+    description: 'Optional comma separated list of coverage reports to send to Codacy'
```

Comment:
Can we have a real yaml list instead a comma separated string?

Reply:
We can't unfortunately. Github actions doesn't support it yet

Reply:
https://github.community/t5/GitHub-Actions/Can-action-inputs-be-arrays/td-p/33776

https://github.com/actions/toolkit/issues/184